### PR TITLE
Removes asstation

### PR DESCRIPTION
**Pull request by pudl**
@CorruptComputer You better come here and give a damning argument as to why this map should stay.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -14,7 +14,6 @@ env:
     - DM_MAPFILE="metastation"
     - DM_MAPFILE="ministation"
     - DM_MAPFILE="dreamstation"
-    - DM_MAPFILE="birdstation"
```
Comment (ChangelingRain) [inline .travis.yml]: no

Comment (pudl) [inline .travis.yml]: Lmao this was an accident

**Comment by TheRealGLH** [comment]
Lavalandstation when?

**Comment by Kromgar** [comment]
Remove efficiency too for the love of god

**Comment by pudl** [comment]
@Kromgar Coiax is apparently maintaining that now.

**Comment by CorruptComputer** [comment]
@pudl ¯\_(ツ)_/¯

**Comment by Bawhoppen** [comment]
@Kromgar we only remove maps if they are dismaintained, not just cause you don't like them, cause everyone has completely different maps they hate for some reason

**Comment by ChangelingRain** [comment]
Asteroid ISN'T maintained, I don't think it actually compiles at the moment, it's not in travis, and you can't select it on the server.

**Comment by Bawhoppen** [comment]
yea I'm saying removing ass is fine cause it isn't maintained and erro was the original maker anyways

**Comment by optimumtact** [comment]
Hahhah here I go again remove fat asses

**Comment by HankMoodyExperience** [comment]
rip favorite map for like the third time

**Comment by optimumtact** [comment]
Like all good ass maps, it'll be back

**Pull request merged**